lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -198,6 +198,14 @@ def migrate(migrations_path, target_version = nil)
         end
       end
 
+      def status(migrations_path, target_version = nil)
+        case
+          when target_version.nil?              then self.new(:up, migrations_path, target_version).status
```

Comment:
I suggest removing the "then" and moving the "then" action to the next line and also using new instead of self.new - they do the same thing.